src/developer-quickstart.md

```diff
@@ -0,0 +1,441 @@
+3. **Modify the App.**
+4. **Run our project.**
+
+### Initialize a React project
+
+To split better our project let's create a new folder `frontend` and initialize our project inside it.
+
+Inside `counter-dapp` run;
+
+```sh
+npx create-react-app frontend --template typescript
+```
+
+The command will generate a react app using [`Create React App`](https://create-react-app.dev/).
+
+#### Install the `fuels` SDK dependencies
+
+On this step we need to install 3 dependencies for the project:
+
+1. `fuels`: The umbrella package that includes all the main tools; `Wallet`, `Contracts`, `Providers` and more.
+2. `fuelchain`: Fuelchain is the ABI TypeScript generator.
+3. `typechain-target-fuels`: The Fuelchain Target is the specific interpreter for the [Fuel ABI Spec](https://github.com/FuelLabs/fuel-specs/blob/master/specs/protocol/abi.md).
+
+> ABI stands for Application Binary Interface. ABI's inform the application the interface to interact with the VM, in other words, they provide info to the APP such as what methods a contract has, what params, types it expects, etc...
+
+##### Install
+
+Inside `counter-dapp/frontend` run;
+
+```sh
+npm install fuels --save
+npm install fuelchain typechain-target-fuels --save-dev
+```
+
+##### Generating contract types
+
+To make it easier to interact with our contract we use `fuelchain` to interpret the output ABI JSON from our contract. This JSON was created on the moment we executed the `forc build` to compile our Sway Contract into binary. If you see the folder `counter_dapp/contract/out` you will be able to see the ABI JSON there. If you want to learn more, read the [ABI Specs here](https://github.com/FuelLabs/fuel-specs/blob/master/specs/protocol/abi.md).
+
+Inside `counter-dapp/frontend` run;
+
+```sh
+npx fuelchain --target=fuels --out-dir=./src/contracts ../fuel-counter/out/debug/*-abi.json
+```
+
+You should see something like this:
+
+```sh
+Successfully generated 4 typings!
+```
+
+Now you should be able to find a new folder `counter-dapp/frontend/src/contracts`. This folder was auto-generated by our `fuelchain` command, this files abstract the work we would need to do to create a contract instance, and generate a complete TypeScript interface to the Contract, making easy to develop.
+
+### Create A Wallet
+
+In the root of the frontend project create a file, createWallet.js
+
+```js
+const { Wallet } = require("fuels");
+
+const wallet = Wallet.generate();
+
+console.log("address", wallet.address.toString());
+console.log("private key", wallet.privateKey);
+```
+
+In a terminal, run the following command: `node createWallet.js`. You should see an output that looks like this:
+
+```console
+address fuel160ek8t7fzz89wzl595yz0rjrgj3xezjp6pujxzt2chn70jrdylus5apcuq
+private key 0x719fb4da652f2bd4ad25ce04f4c2e491926605b40e5475a80551be68d57e0fcb
+```
+
+Copy the private key, you will need this later to set it as a string value for a variable `WALLET_SECRET` in your `App.tsx` file. More on that below.
```

Comment:
Maybe instead of copying, we could say to the user to save these in a separate file to use in the next steps.